recipe/meta.yaml

```diff
@@ -143,6 +144,8 @@ outputs:
     test:
       files:
         - test_data
+        - run_test_lib.sh  # [unix]
+        - run_test_lib.bat  # [win]
```

Comment:
My understanding is that the `run_test` would be run automatically without the need to list it. But not sure if they are run twice or only once with both packages installed. Either way I don't think we need this change. @msarahan probably knows more.

Reply:
Hmm it doesn't seem to be here (or in #229). Well I can't find the outputs of these tests in the log (https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/run_test.sh#L7 and https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/run_test.py#L13 should print something) though I'm sure I'm missing something.

BTW if these tests were run we would have caught the missing driver as `run_test.py` checks for it...

Reply:
Yep. You are, obviously, correct :grimacing: 

@msarahan I guess that it is a bug in conda-build, no? What is the recommend way to run tests for split packages in an integrated way? Or separated?

@gillins meanwhile let's merge #229 but let's keep working on this one to get the tests back up.